alerting/alert/type.go

```diff
@@ -17,6 +17,9 @@ const (
 	// TypeGitHub is the Type for the github alerting provider
 	TypeGitHub Type = "github"
 
+	// TypeGitlab is the Type for the github alerting provider
+	TypeGitlab Type = "gitlab"
```

Comment:
Another small nitpick - this should technically be `TypeGitLab` since the `L` is capitalized in GitLab.

Reply:
Indeed it's better. All done